pal/src/include/pal/virtual.h

```diff
@@ -105,108 +105,6 @@ BOOL VIRTUALOwnedRegion( IN UINT_PTR address );
 
 #ifdef __cplusplus
 }
-
-/*++
-Class:
-    ExecutableMemoryAllocator
-
-    This class implements a virtual memory allocator for JIT'ed code.
-    The purpose of this allocator is to opportunistically reserve a chunk of virtual memory
-    that is located near the coreclr library (within 2GB range) that can be later used by
```

Comment:
Although we don't do this now because it is a hassle to do it in Windows, may be can make use of this in our JIT'ed code as well?